### ISSUE TITLE
Can you clean up the repo when you have time?

### ISSUE THREAD

**Issue by stepjames1998** [limanling/clip-event#7]

**Comment by stepjames1998** [comment]
The repo is currently in such a mess. It is hard to follow this work.

**Comment by BudBudding** [comment]
I have been trying to reproduce this work recently, but it has not been successful and I have encountered many problems.

**Comment by stepjames1998** [comment]
As the repo of a top conference paper, this is so unclear.

**Comment by HXYM** [comment]
Much of the code provided by the research she was engaged in was missing things, delayed for a long time, and was ready for a different angle.

**Comment by IceIce1ce** [comment]
+1
